instrumentation-api-incubator/src/main/jflex/SqlSanitizer.jflex

```diff
@@ -5,6 +5,8 @@
 
 package io.opentelemetry.instrumentation.api.incubator.semconv.db;
 
+import java.util.HashMap;
```

Comment:
I think you should do anything with the sanitizer. If you wish to keep the literals that the sanitizer removes you should just disable the sanitizer.

Reply:
Fine by me. For consistency purpose, how should I handle users having sanitizer + prepared statements? 

Such as 
```sql
SELECT 1 FROM table WHERE id=?
// statement.set(0, 'id');
```

This, without sanitizer, would yield
```
db.query: SELECT 1 FROM table WHERE id=?
db.query.parameter.0: 'id'
```
which makes sense.

But with the sanitizer it would be
```
db.query: SELECT ? FROM table WHERE id=?
db.query.parameter.0: 'id'
```
which, for me, looks a little bit weird. On this example it's pretty straightforward, but on complex queries I feel like it can become difficult to understand what's happening. Should I just don't care and ignore this case?

Reply:
since capturing the parameters and sanitizer have conflicting goals we could consider disabling the sanitizer when capturing parameters is enabled

Reply:
this makes sense to me for now, prior to stable database semconv

Reply:
Thanks both of you. I didn't remember your comment @trask , sorry for this.